api/lib/source/cosmosV2-source.js

```diff
@@ -6,8 +6,8 @@ class CosmosV2API extends CosmosV0API {
     this.reducers = require('../reducers/cosmosV2-reducers')
   }
 
-  async query(url, resultSelector = 'result') {
-    const response = await this.getRetry(url)
+  async query(url, intent = 0, resultSelector = 'result') {
```

Comment:
in cosmosV0-source the argument is `noRetry` - can it be the same here?

Reply:
yes, absolutely. You are right. Better stay consistent

Reply:
although I see. I need to change and add the noRetry logic here too. Ah, thanks for catching this